src/tools/crossgen2/ILCompiler.DependencyAnalysisFramework/PerfEventSource.cs

```diff
@@ -10,22 +11,49 @@
 namespace ILCompiler.DependencyAnalysisFramework
 {
     // The event IDs here must not collide with the ones used by ReadyToRunPerfEventSource.cs
```

Comment:
This comment applies to PerfEventSource, not StartStopEvents.

Reply:
Moved this down to the actual events.